doc/releases/1.15.1dev.rst

```diff
@@ -10,15 +10,23 @@ Dependency Changes
 
 Functionality/Performance Improvements and Additions
 ----------------------------------------------------
+<<<<<<< HEAD
+- Allow the selection of a secondary filter option  for all spectrographs
+that need it; see new ``filter2`` parameter.
+=======
 - When dealing with `bkg_redux` reduction (i.e., A-B), a sky model (called `bkg_redux_skymodel`)
   is generated using the A frame, while the existing `skymodel` is generated using the A-B frame.
   This allows to have an actual 1D extracted sky spectrum and to perform the flexure correction for
   `bkg_redux` reduction.
 - Various improvements in the flexure correction and added the possibility to use a modeled archive
   sky spectrum generated with `pypeit.wavemodel.nearIR_modelsky()` to perform the flexure correction.
+>>>>>>> 5dbd2b1a78748a2d947b8c2e60b9e0de4543589d
```

Comment:
It looks like you had a merge conflict.  Can you please remove lines 13, 16, and 23?

Reply:
Hi @kbwestfall 

Thank you for your continued guidance and feedback. I've addressed the merge conflict as per your instructions and removed the lines 13, 16, and 23.

Please let me know if everything looks good now or if there's anything else you need from my end.

Appreciate your help in getting this PR ready for merging!